src/app/about-us/mission-vision/mission-vision.component.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { TranslateModule, TranslateLoader, TranslateFakeLoader } from '@ngx-translate/core';
+import { MissionVisionComponent } from './mission-vision.component';
```

Comment:
Please separate your imports with a blank line according to this:
angular imports

external imports

internal imports

It would be something like this:
```
import { async, ComponentFixture, TestBed } from '@angular/core/testing';

import { TranslateModule, TranslateLoader, TranslateFakeLoader } from '@ngx-translate/core';

import { MissionVisionComponent } from './mission-vision.component';
```